sci-astronomy/wcstools/wcstools-3.9.5.recipe

```diff
@@ -0,0 +1,183 @@
+	cmd:gettab$commandSuffix = $portVersionCompat
+	cmd:i2f$commandSuffix = $portVersionCompat
+	cmd:imcat$commandSuffix = $portVersionCompat
+	cmd:imextract$commandSuffix = $portVersionCompat
+	cmd:imfill$commandSuffix = $portVersionCompat
+	cmd:imhead$commandSuffix = $portVersionCompat
+	cmd:immatch$commandSuffix = $portVersionCompat
+	cmd:imresize$commandSuffix = $portVersionCompat
+	cmd:imrot$commandSuffix = $portVersionCompat
+	cmd:imsize$commandSuffix = $portVersionCompat
+	cmd:imsmooth$commandSuffix = $portVersionCompat
+	cmd:imstack$commandSuffix = $portVersionCompat
+	cmd:imstar$commandSuffix = $portVersionCompat
+	cmd:imwcs$commandSuffix = $portVersionCompat
+	cmd:isfile$commandSuffix = $portVersionCompat
+	cmd:isfits$commandSuffix = $portVersionCompat
+	cmd:isnum$commandSuffix = $portVersionCompat
+	cmd:isrange$commandSuffix = $portVersionCompat
+	cmd:keyhead$commandSuffix = $portVersionCompat
+	cmd:newfits$commandSuffix = $portVersionCompat
+	cmd:scat$commandSuffix = $portVersionCompat
+	cmd:sethead$commandSuffix = $portVersionCompat
+	cmd:setpix$commandSuffix = $portVersionCompat
+	cmd:simpos$commandSuffix = $portVersionCompat
+	cmd:sky2xy$commandSuffix = $portVersionCompat
+	cmd:skycoor$commandSuffix = $portVersionCompat
+	cmd:sp2char$commandSuffix = $portVersionCompat
+	cmd:subpix$commandSuffix = $portVersionCompat
+	cmd:sumpix$commandSuffix = $portVersionCompat
+	cmd:wcscrlf$commandSuffix = $portVersionCompat
+	cmd:wcsgetdate$commandSuffix = $portVersionCompat
+	cmd:wcshead$commandSuffix = $portVersionCompat
+	cmd:wcsremap$commandSuffix = $portVersionCompat
+	cmd:wcstools$commandSuffix = $portVersionCompat
+	cmd:xy2sky$commandSuffix = $portVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	wcstools${secondaryArchSuffix}_devel = $portVersionCompat
+	devel:libwcstools$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	wcstools$secondaryArchSuffix == $portVersion base
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:sed
+	"
+
+TEST_REQUIRES="
+	cmd:test
+	"
+
+defineDebugInfoPackage wcstools$secondaryArchSuffix \
+	"$libDir"/libwcstools.so.$libVersion
+
+PATCH()
+{
+	sed -i \
+		-e 's/getdate/wcsgetdate/' \
+		-e 's/crlf/wcscrlf/' \
+		-e 's/remap/wcsremap/' \
+		-e "s/3.... Programs/${portVersion} Programs/" \
+		wcstools
+}
+
+BUILD()
+{
+	for f in "$portDir"/additional-files/{configure.ac,wcstools.pc.in,Makefile.am}; do
+		cp ${f} .
+	done
+	cp "$portDir"/additional-files/Makefile.libwcs.am libwcs/Makefile.am
+
+	mkdir -p m4
+
+	libtoolize --force --copy --install
+	aclocal --force --install
+	autoheader --force
+	autoconf --force
+	automake --add-missing --force-missing
+
+	runConfigure --omit-dirs binDir ./configure --bindir="$commandBinDir"
```

Comment:
```
LDFLAGS="-lnetwork" runConfigure --omit-dirs binDir ./configure --bindir="$commandBinDir"
```
works for me.

Reply:
Actually I opted for a bit more love in configure.ac